perf/README.md

```diff
@@ -45,20 +45,35 @@ Rest of the instructions assume the name is `settings.env`
 |WORKLOAD| One of the workloads in the `workload` directory|
 |TOPIC_PREFIX|Prefix for kafka topic. Default is `dbserver1`|
 |UID| uid that docker processes should use. Get from `id -u ${USER}`|
+|SNAPSHOT_MODE| Whether to use snapshot mode or not|
 
-##  Start all services in docker
+##  Running with CDCSDK
 
 ### Start CDCSDK Server with NullChangeConsumer
 
     # Start CDCSDK Server
-    docker compose -f cdcsdk-base.yaml -f cdcsdk-null.yaml --env-file settings.env up -d
+    docker compose -f cdcsdk-base.yaml -f cdcsdk-null.yaml --env-file <path to>/settings.env -d
 
 ### Start CDCSDK Server with Kafka + PG on local system.
 
-    docker compose -f sinks/kafka/confluent.yanl --env-file settings.env up -d
-    docker compose -f sinks/kafka/kafka-pg.yaml --env-file settings.env up -d
+    docker compose -f sinks/kafka/confluent.yaml --env-file <path to>/settings.env up -d
+    docker compose -f sinks/kafka/kafka-connect-pg.yaml --env-file <path to>/settings.env up -d
     docker compose -f cdcsdk-base.yaml -f cdcsdk-kafka-local.yaml --env-file \
-        settings.env up -d
+        <path to>/settings.env up -d
+
+##  Running with Kafka-Connect
+
+### Go to the directory yb-connect
+
+    cd yb-connect
```

Comment:
Maybe this doesn't work. This statement suggests change the directory but the paths to confluent.yaml is not correct. Maybe we give instructions from `perf` directory only?

Reply:
Alright, making appropriate changes